cmd/kubelet/app/server.go

```diff
@@ -651,6 +653,7 @@ type KubeletConfig struct {
 	Auth                           server.AuthInterface
 	Builder                        KubeletBuilder
 	CAdvisorInterface              cadvisor.Interface
+	CalculateVolumeDiskUsageFrequency time.Duration
```

Comment:
I know it's the convention in this struct, but I really don't think a "Frequency" should have a "Duration" unit...